scripts/circle-ci/run-circle-tests.sh

```diff
@@ -52,14 +52,14 @@ done
 
 # Short circuit the build if it's docs only
 if ./scripts/circle-ci/check-only-docs-changes.sh; then
-    if [ $CIRCLE_NODE_INDEX -eq 0 ]; then
+    if [ $1 -eq 0 ]; then
```

Comment:
while I agree that your change means that we reference the first arg of this script as opposed to the env variable, I think it's preferable to declare a variable in the script and then use it in named form elsewhere

Reply:
fixed